src/pages/SetPasswordPage.js

```diff
@@ -82,10 +85,23 @@ class SetPasswordPage extends Component {
         super(props);
 
         this.validateAndSubmitForm = this.validateAndSubmitForm.bind(this);
+        this.isFormValid = this.isFormValid.bind(this);
+        this.validatePassword = this.validatePassword.bind(this);
+        this.validateConfirmPassword = this.validateConfirmPassword.bind(this);
```

Comment:
Why is this method bound to this? It doesn't look like it needs to be (there are probably other methods like this too)